mobile/Screens/PasswordResetScreen.jsx

```diff
@@ -0,0 +1,187 @@
+      return;
+    }
+    updateHashedCode(codeData);
+    setStep('verifyCode');
+    setErrorMsg('');
+  };
+
+  // Check if the user inputted code is correct
+  const verifyCode = async () => {
+    const data = await axios.post(`${process.env.EXPO_PUBLIC_SERVER_URL}/passwordReset/verifyCode`, { code, hashedCode });
+    const match = data.data;
+    if (match) {
+      setStep('inputNewPassword');
+      setErrorMsg('');
+      return;
+    }
+    // Banner if code doesn't match, have some indicator
+    // else we indicate to user that code doesn't match, retry
+    setErrorMsg('Invalid Code');
+  };
+
+  // Updates the user's password
+  // Outputs an error message otherwise
+  const updatePassword = async () => {
+    if (confirmation !== newPassword) {
+      setErrorMsg('Confirmation must match the new password.');
+      return;
+    }
+    if (newPassword.length < 5) {
+      setErrorMsg('Password must be at least 5 characters.');
+      return;
+    }
+    const data = await axios.post(`${process.env.EXPO_PUBLIC_SERVER_URL}/passwordReset/updatePassword`, { email, newPassword });
+    redirectLogIn();
+    setErrorMsg('');
+  };
+
+  // Controls the display based on which step the user is on in the password reset flow
+  const updateDisplay = () => {
+    let view = [<View />];
+    if (step === 'getVerification') {
+      view = [
+        <View>
+          <Button title="Back" onPress={redirectLogIn} />
+        </View>,
+        <View style={styles.inputContainer}>
+          <TextInput
+            style={styles.textfields}
+            onChangeText={handleChangeEmail}
+            value={email}
+            placeholder="Email"
+            placeholderTextColor="#000000"
+            autoCapitalize="none"
+          />
+        </View>,
+        <Button title="Submit" onPress={sendPasswordReset} />,
+      ];
+    } else if (step === 'verifyCode') {
+      view = [
+        <View>
+          <Button title="Back" onPress={() => { setStep('getVerification'); }} />
+        </View>,
+        <View style={styles.inputContainer}>
+          <TextInput
+            style={styles.textfields}
+            onChangeText={handleChangeCode}
+            placeholder="Verification Code"
+            placeholderTextColor="#000000"
+            value={code}
+          />
+        </View>,
+        <Button title="Get Another Code" onPress={sendPasswordReset} />,
+        <Button title="Check Code" onPress={verifyCode} />,
+      ];
+    } else if (step === 'inputNewPassword') {
+      view = [
+        <View>
+          <Button title="Cancel" onPress={redirectLogIn} />
+        </View>,
+        <View style={styles.inputContainer}>
+          <TextInput
+            style={styles.textfields}
+            onChangeText={setNewPassword}
+            placeholder="New Password"
+            placeholderTextColor="#000000"
+            autoCapitalize="none"
+            value={newPassword}
+            secureTextEntry="true"
```

Comment:
We can pass secureTextEntry without the `="true"` since secureTextEntry is a bool not a string! The attribute will be assigned a value of true by default.